prediction/config.py

```diff
@@ -4,17 +4,16 @@
 
     Provides the flask config options
 """
-import glob
 import os
 
 
 class Config(object):
     PROD_SERVER = os.getenv('PRODUCTION', False)
     DEBUG = False
     SEGMENT_ASSETS_DIR = os.path.abspath(os.path.join(os.getcwd(), 'src', 'algorithms', 'segment', 'assets'))
-    DICOM_PATHS_DOCKER = glob.glob(os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**'))
-    DICOM_PATHS_LOCAL = glob.glob(
-        os.path.join(os.getcwd(), 'src', 'tests', 'assets', 'test_image_data', 'full', 'LIDC-IDRI-*', '**', '**'))
+    DICOM_PATHS_DOCKER_WILDCARD = os.path.join('/images_full', 'LIDC-IDRI-*', '**', '**')
+    DICOM_PATHS_LOCAL_WILDCARD = os.path.join(os.getcwd(), 'src', 'tests', 'assets', 'test_image_data', 'full',
```

Comment:
I'd juuust add a brief comment along the lines of "these are expanded at runtime" so that nobody gets too confused.

(Also I don't quite like the use of ``getcwd`` - is that "fixed"? Seems more sensible to use underscoreunderscorefileunderscoreunderscore)

Reply:
You have to be in the `prediction` directory otherwise none of your code imports for the prediction code would work since this directory is copied into the prediction docker container. However, I can use `__file__` here as well.

Reply:
> You have to be in the prediction directory

Lets fix that another time, but let's at least not make the situation "worse" ;)